backend/agentd/middlewares.go

```diff
@@ -40,12 +40,6 @@ func agentLimit(next http.Handler) http.Handler {
 	return AgentLimiterMiddleware(next)
 }
 
-// entityLimit is the abstraction layer required to be able to change at
-// runtime the actual function assigned to EntityLimiterMiddleware above.
-func entityLimit(next http.Handler) http.Handler {
-	return EntityLimiterMiddleware(next)
```

Comment:
And we don't want to also clean up `EntityLimiterMiddleware`?

Reply:
nvm, I see the other references.

Reply:
Better to leave the exported symbol there to avoid build-breakage downstream.